.buildkite/e2e/release-branch-matrix.yaml

```diff
@@ -65,7 +65,7 @@
   fixed:
     E2E_PROVIDER: ocp
   mixed:
-    - DEPLOYER_CLIENT_VERSION: "4.19.2"
+    - DEPLOYER_CLIENT_VERSION: "4.20.1"
```

Comment:
I think 4.20.4 is available: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.20.4/